src/link/style.scss

```diff
@@ -301,6 +301,19 @@ body {
 				animation-name: wikipediapreview-edit-inline-loader-expanded;
 			}
 		}
+
+		&-tooltip {
+			width: 190px;
+			height: 40px;
+			background: #36c;
+			color: #fff;
+			display: flex;
+			justify-content: center;
+
+			&-text {
+				align-self: center;
+			}
+		}
```

Comment:
<img width="553" alt="image" src="https://github.com/wikimedia/wikipediapreview-wordpress/assets/2560096/976f1313-d492-4e97-8956-4f904eaf4379">
we also need some offset for mobile UI

Reply:
Done 
<img width="512" alt="image" src="https://github.com/wikimedia/wikipediapreview-wordpress/assets/4752599/51f330e4-6b8c-4592-bd6a-bfe102e84c4e">